packages/plugins/debug-renderer/src/index.js

```diff
@@ -0,0 +1,422 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Visualizer for debugging custom renderers in Blockly.
+ * @author fenichel@google.com (Rachel Fenichel)
+ */
+import * as Blockly from 'blockly';
```

Comment:
Don't import all of Blockly. This imports blocks, JS generator, etc. 
For plugins, we just want to import core. 

```
import Blockly from 'blockly/core';
```